src/ixdat/techniques/ms.py

```diff
@@ -500,12 +528,126 @@ def gas_flux_calibration_curve(
             n_dot_fit = np.array([0, max(n_dot_vec)])
             S_fit = n_dot_fit * pfit[0] + pfit[1]
             ax.plot(n_dot_fit * 1e9, S_fit * 1e9, "--", color=color)
-        cal = CalPoint(mol=mol, mass=mass, F_type=cal_type, F=F, date=self.yyMdd)
+        cal = MSCalResult(
+            name=f"{mol}@{mass}",
+            mol=mol,
+            mass=mass,
+            cal_type=cal_type,
+            F=F,
+        )
+        if return_ax:
+            return cal, ax
+        return cal
+
+    def siq_gas_flux_calibration(self, mol, mass, tspan, chip=None):
+        """Simple pure-gas flux calibration, using external MS quantification package
```

Comment:
it is a highly specialised external package right? Or would this function be used with any external MS quantification pacakge

Reply:
Good point. "external MS quantification package" replaced with "`spectro_inlets_quantification`" here and elsewhere.